minio/api.py

```diff
@@ -2769,15 +2769,17 @@ def upload_snowball_objects(self, bucket_name, object_list, metadata=None,
         else:
             length = os.stat(name).st_size
 
+        part_size = max(length, 5 * (1024) ** 2)
```

Comment:
This fix is `part_size = 0 if length < MIN_PART_SIZE else length`

Reply:
Updated the PR accordingly.